File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        li=[]
        for i  in range(0,len(nums)):
            a=target-nums[i]
            if a in nums :
                ind=nums.index(a)
                if ind!=i:
                    li.append(i)
                    li.append(ind)
                    return li
        return li

        

File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> mp;
        int n=nums.size();
        
        for(int i=0;i<n;i++){
            int a=target-nums[i];
            if(mp.count(a)){
                return {i,mp[a]};
            }
            mp[nums[i]]=i;
        }
        return {0,0};
    }
};